src/Workspaces/Core/Portable/Utilities/EditDistance.cs

```diff
@@ -211,16 +564,16 @@ public static bool IsCloseMatch(string originalText, string candidateText, int c
                 // other matches).
                 if (candidateText.IndexOf(originalText, StringComparison.OrdinalIgnoreCase) >= 0)
                 {
-                    matchCost = costThreshold;
+                    matchCost = threshold;
                 }
             }
 
-            if (matchCost > costThreshold)
+            if (matchCost > threshold)
             {
                 return false;
             }
 
-            matchCost += Penalty(candidateText, originalText);
+            matchCost += Penalty(candidateText, this.originalText);
```

Comment:
There's a bug in Penalty where lengthDifference of 0 and 1 are treated the same. Should we file that separately?